pom.xml

```diff
@@ -198,14 +204,14 @@
             <id>sonatype-snapshots</id>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
         </repository>
-        <repository>
+        <!-- <repository>
             <id>de.tudarmstadt.ukp.wikipedia</id>
             <url>http://zoidberg.ukp.informatik.tu-darmstadt.de/artifactory/public-snapshots/</url>
             <snapshots>
                 <enabled>true</enabled>
                 <updatePolicy>always</updatePolicy>
             </snapshots>
-        </repository>
+        </repository> -->
```

Comment:
Delete not comment out?

Reply:
Rather comment out at the moment, because we are gonna re enable that source when they release the snapshot (that's why I've added the 'TODO' at the top).